command/server/params.go

```diff
@@ -185,8 +184,7 @@ func (p *serverParams) generateConfig() *server.Config {
 		LogLevel:           hclog.LevelFromString(p.rawConfig.LogLevel),
 		JSONLogFormat:      p.rawConfig.JSONLogFormat,
 		LogFilePath:        p.logFileLocation,
-		TLSCertFile:        p.rawConfig.TLSCertFile,
-		TLSKeyFile:         p.rawConfig.TLSKeyFile,
+		UseTLS:             p.rawConfig.UseTLS,
```

Comment:
Just thinking out loud:
if this config parameter is related only for the JSON RPC, would it make sense putting it within the `JSONRPC` config section (alongside with `JSONRPCAddr`, `AccessControlAllowOrigin` etc.)?

On the other hand, if it is not strictly related to the json rpc, leave it here.

Reply:
For now it is only JSON related and most likely will remain only for JSON, but in theory TLS can be used for any additional endpoint, for instance GRPC, although at this moment there is no need for that.